src/Components/Programs.jsx

```diff
@@ -41,8 +41,8 @@ function Programs() {
             </Row>
             <Row className="justify-content-center programs" id="thePrograms">
                 <Col className="firstProgramText" md="8">
-                    {data.vissionOnServicesPageCollection.items[0] ? (
-                        data.vissionOnServicesPageCollection.items.map(vision => (
+                    {data.visionOnServicesPageCollection.items[0] ? (
+                        data.visionOnServicesPageCollection.items.map(vision => (
                             <ReactMarkdown key={vision.vission} className="programInfoSection">
                                 {vision.vission}
```

Comment:
Do these vissions need to be updated as well?

Reply:
No, the key stayed the same (I wasn't able to edit it without needing to redo the contentful model)